### PR TITLE
Improve troubleshoot tags script and adjust tags

### PR DESCRIPTION
**Pull request by PedroAntunesCosta**

### CODE REVIEW

File: docs-utils/troubleshooting-structure-adjustment.js [hunk 1]
```diff
@@ -11,12 +11,21 @@ function moveArticleTagsToFrontmatter(filePath) {
     if (fileContent.includes('**Tags:**')) {
         const tags = fileContent.split('**Tags:**')[1].split('\n')[0]
```
Comment (julia-rabello) [inline docs-utils/troubleshooting-structure-adjustment.js:12]:
```suggestion
        const tags = fileContent.split('**Tags:**')[1].split('\n')[0].trim()
```

Comment (julia-rabello) [reply docs-utils/troubleshooting-structure-adjustment.js:12]: There is whitespace before and after some tags, like in docs/en/troubleshooting/Integrations/why-dont-marketplace-orders-integrate-with-my-store.md

I think adding a trim here would fix it

File: docs/en/troubleshooting/Store operations/i-cant-index-a-product-in-the-catalog.md [hunk 2]
```diff
@@ -13,10 +13,9 @@ slugEN: i-cant-index-a-product-in-the-catalog
 locale: en
 legacySlug: i-cant-index-a-product-in-the-catalog
 subcategoryId: 2Q0IQjRcOqSgJTh6wRHVMB
+tags: : Catalog, Index, Product, SKU, Trade policy
 ---
 
-**Tags:**: Catalog, Index, Product, SKU, Trade policy
```
Comment (julia-rabello) [inline docs/en/troubleshooting/Store operations/i-cant-index-a-product-in-the-catalog.md:18]: There is an extra `:` in line 16 only in this file, probably introduced by a double `:` coming from the original tags text (previously on line 18)

**Review by julia-rabello — CHANGES_REQUESTED** [review]

File: docs/en/troubleshooting/Store operations/i-cant-index-a-product-in-the-catalog.md (hunk 2 above)
Comment (PedroAntunesCosta) [reply docs/en/troubleshooting/Store operations/i-cant-index-a-product-in-the-catalog.md]: Good catch! I'll fix it in Contentful as well.

**Review by julia-rabello — APPROVED** [review]

**Pull request merged**